tests/calloc_test.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <aws/common/common.h>
+#include <aws/common/math.h>
+#include <aws/testing/aws_test_harness.h>
+
+#ifdef __MACH__
+#    include <CoreFoundation/CoreFoundation.h>
```

Comment:
What is this needed for?

Reply:
It was in the realloc test file, but you're right its not needed here.  Removed.